src/main/java/org/myrobotlab/service/HtmlFilter.java

```diff
@@ -197,29 +196,6 @@ public void attachTextListener(String name) {
     addListener("publishText", name);
   }
 
-  @Override

```

Comment:
This service does actually have configuration options..  why was this removed?  this should be updated to suppport those config options.. not removed.

Reply:
it still has listener, it only had that ... same as all services

Reply:
The HtmlFilterConfig specifies  .. those are not being honored here. 
public String preHtmlTag = "<pre>";
public String postHtmlTag = "</pre>";